bundle/jsky.app.ot/src/main/scala/jsky/app/ot/editor/template/TemplateParametersEditor.scala

```diff
@@ -309,7 +309,7 @@ class TemplateParametersEditor(shells: java.util.List[ISPTemplateParameters]) ex
 
     object MagnitudesPanel extends ColumnPanel {
       def magRow(band: Magnitude.Band): Row = {
-        lazy val zero = new Magnitude(band, 0.0)
+        lazy val zero = new Magnitude(band, 0.0, band.defaultSystem)
```

Comment:
We can go back and remove the extra argument now with your update to the `Magnitude` constructors.  It's fine either way though.